.github/workflows/main.yml

```diff
@@ -15,6 +17,8 @@ jobs:
           - 14
     steps:
       - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
```

Comment:
I don't think millions of repos should have to set this manually. The default should be changed in https://github.com/actions/checkout/pull/946